Bio/Blast/Record.py

```diff
@@ -74,7 +74,7 @@ def database_letters(self):
             "version of Biopython .",
             BiopythonDeprecationWarning)
 
-        return self._database_letters
+        return self.database_letters
```

Comment:
This change defines the property ``.database_letters`` in terms of itself, which is likely to cause a recursion error (or some other error - I'm curious to see exactly what will happen).